client/src/router/index.js

```diff
@@ -64,7 +64,7 @@ const routes = [
                 children: [
                     // There must be routes which need logged user
                     {
-                        path: '',
+                        path: '/',
```

Comment:
What does it fix?

Reply:
there was an error that a slash was needed, I added